pkg/motor/motor.go

```diff
@@ -65,15 +66,22 @@ type motorNodeImpl struct {
 	Resources *motorResources
 }
 
-func EmptyMotor(id string) *motorNodeImpl {
+func EmptyMotor(id string, logLevel string) *motorNodeImpl {
```

Comment:
After this is updated to `dev` you could add log level as a property in the `InitializeRequest` [protobuf file](https://github.com/sonr-io/sonr/blob/dev/proto/motor/v1/request.proto). This would keep it inline with existing method.